blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
+- `uiState`: String; css defined for `is-selected`, `is-updated` (default), `is-reviewed` in Melody
```

Comment:
Oh, just saw the status field a bit later. Hmm, it feels like `uiState` is combining two separate notions that the component should be responsible for dealing with, specifically:

* whether or not it's selected (which feels like UI state)
* whether or not it's been updated/reviewed/etc (which feels like domain logic)